Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Core/BindingContract.cs

```diff
@@ -42,12 +44,12 @@ public sealed class BindingContract : Entity
         DateTimeOffset bindingFrom,
         DateTimeOffset expiringAt) => new(id, customerId, duration, bindingFrom, expiringAt);
 
-    public Annex AddAnnex(DateTimeOffset validFrom, DateTimeOffset now)
+    public void AddAnnex(DateTimeOffset validFrom, DateTimeOffset now)
     {
         BusinessRuleValidator.Validate(
             new AnnexCanOnlyBeAddedOnlyBeAddedToActiveBindingContractRule(TerminatedAt, ExpiringAt, now));
 
-        return Annex.Add(Id, validFrom);
+        RegisteredAnnexes.Add(Annex.Add(Id, validFrom));
```

Comment:
We could change RegisteredAnnexes to be similar name to method that is defining it.

var annex = Annex.Create(Id, validFrom);
AttachedAnnexes.Add(annex);

What you think?

Reply:
Done!